src/ostream

```diff
@@ -27,6 +27,11 @@
 #include "cstdio"
 #include "ostream_helpers"
 
+#ifdef ARDUINO_ARCH_AVR
+	#include <avr/pgmspace.h> // typedef PGM_P, pgm_read_byte()
+	#include "WString.h"	  // class __FlashStringHelper
```

Comment:
Probably better to just `#include <Arduino.h>`, which should include both of these in a more portable way (at the expense of incuding more than maybe needed).